formbricks/formbricks
# fix: Multiple Choice Question's Other Field submits each value onChange rather than the final Value

## What does this PR do?

- fixes Multiple Choice Question's Other Field submits each value onChange rather than the final Value issue

Fixes https://github.com/formbricks/formbricks/issues/1028



https://github.com/formbricks/formbricks/assets/52622674/7f6575d0-4be3-4fa8-aabe-9deb105c6170





## Type of change




## How should this be tested?

Create a Survey with Multiple Choice question and add an "Other" option type and Publish the survey
Now open the survey, click on the other option and fill anything and click submit
Now go to the survey response page and view how the response is shown.

## Checklist

### Required


### Appreciated
